app/models/health_card.rb

```diff
@@ -0,0 +1,9 @@
+class HealthCard
+  include ActiveModel::Model
```

Comment:
Don't know, and it seems to work fine without it so will remove.

Reply:
It's not clear to me why we have all these models in here. If this is just for presentation, these objects seem like they're doing a lot of work if we're just trying to make the FHIR/HC content more accessible to the frontend. It would make more sense to have some kind of composition of existing FHIR resources and/or HealthCard models. If we find we're doing a lot to make the health_cards library workable, the changes need to be made there and not here.